azure-pipelines.yml

```diff
@@ -42,7 +42,8 @@ jobs:
     displayName: Docker setup and build
 
   - publish: $(Build.ArtifactStagingDirectory)
-    condition: eq(variables['COMPILER'], 'gcc')
+    condition: and(succeeded(), eq(variables['COMPILER'], 'gcc'))
+    artifact: RPCS3 for Linux ($(echo $Build.SourceVersion | cut -b 1-8 -f1))
```

Comment:
I wish, but you can only run scripts in script tags. If you want t get specific information, modify it in a setup script like is done here: https://github.com/RPCS3/rpcs3/blob/ebbf329b6aea41a42daf96902b4f19375c05c67d/.ci/setup-windows.sh#L114

Reply:
My previous version, without ```cut``` worked perfectly fine, but title is too long. I think I've missed something if docs. Will fix later.

Reply:
Yes, you've missed that this isn't bash. The YAML can take some weird vagaries that look like powershell syntax. Azure runs inside windows, and the only way to get bash is to use the bash tag.

Reply:
I can finally completely remove commit-id and everything will be fine. Still curious, why we have to recompile the whole project for 20-30 mins, to just check the name.